src/main/scala/io/iohk/ethereum/consensus/ethash/difficulty/ConfiguredDifficultyCalculator.scala

```diff
@@ -0,0 +1,36 @@
+package io.iohk.ethereum.consensus.ethash.difficulty
+
+import io.iohk.ethereum.consensus.difficulty.DifficultyCalculator
+import io.iohk.ethereum.domain.BlockHeader
+
+class ConfiguredDifficultyCalculator(powTargetTime: Long) extends DifficultyCalculator {
+
+  val DifficultyBoundDivision: Int = 2048
+  val FrontierTimestampDiffLimit: Int = -99
+  val MinimumDifficulty: BigInt = 131072
+
+  /**
+    * The LowerBoundExpectedRatio (l for abbreviation below) divides the timestamp diff into ranges:
+    *   [0, l)   => c = 1, difficulty increases
+    *   [l, 2*l) => c = 0. difficulty stays the same
+    *   ...
+    *   [l*i, l*(i+1) ) => c = 1-i, difficulty decreases
+    *
+    *  example:
+    *  powTargetTime := 60 seconds
+    *  l := 30 seconds
+    *   [0, 0.5 min)         => difficulty increases
+    *   [0.5 min, 1.5 min)   => difficulty stays the same (the average should be powTargetTime)
+    *   [1.5 min, +infinity) => difficulty decreases
+    */
+  val LowerBoundExpectedRatio: Long = powTargetTime / 2
```

Comment:
thinking about it now, I did too simple assumption here. WDYT guts, should we configure also the deviation of target time or we should assume that deviation is X % of target time?

Reply:
Hmm I'm not sure if this deviation will bother us too much, neither if it's easy to change this formulas without screwing up... I wouldn't mind leaving it as is for now